nginx/nginx.conf

```diff
@@ -79,20 +79,20 @@ http {
         # Enable CORS support for the font-awesome webfont
```

Comment:
This file still needs to be taken out of the PR (with the exception of the lines specific to password reset).